### PR TITLE
moved login handler to separate file

### PR DESCRIPTION
**Pull request by ronald-so**
Just a quick fix - moved login controller code from `app.py` to `api/login_handler.py`.

### CODE REVIEW

File: server/api/login_handler.py [hunk 1]
```diff
@@ -1,7 +1,64 @@
...
+import jwt
+import datetime
+
 login_handler = Blueprint('login_handler', __name__)
 
 
-@login_handler.route('/login')
+# Loading sample_users.json as a very simple "database"
+with open('sample_users.json') as sample_users:
+    users = json.load(sample_users)
```
Comment (bonnieli) [inline server/api/login_handler.py:11]: wait why is this needed? can you query your Users instead?

Comment (ronald-so) [reply server/api/login_handler.py:11]: definitely! was just part of some really old code, before backend was set up
